ci/infra/jobs/build_amis/dw-al2-concourse-ami.yml

```diff
@@ -0,0 +1,92 @@
+resources:
+  - name: dw-al2-concourse-ami
```

Comment:
slightly confusing names as they are code repos, not AMIs

Reply:
FIxed.